src/tdastro/astro_utils/passbands.py

```diff
@@ -102,134 +168,149 @@ class Passband:
 
     Attributes
     ----------
-    label : str
-        The label of the passband. Note that this is not unique across surveys.
     survey : str
-        The survey to which the passband belongs.
+        The survey to which the passband belongs: eg, "LSST".
+    filter_name : str
+        The name of the passband's filter: eg, "u".
     full_name : str
-        The full name of the passband.
+        The full name of the passband. This is the survey and filter concatenated: eg, "LSST_u".
     table_path : str
         The path to the transmission table file.
     table_url : str
         The URL to download the transmission table file.
-    transmission_table : np.ndarray
+    waves : np.ndarray
+        The wavelengths of the transmission table. To be used when evaluating models to generate fluxes
+        that will be passed to fluxes_to_bandflux.
+    _loaded_table : np.ndarray
+        A 2D array of wavelengths and transmissions. This is the table loaded from the file, and is neither
+        interpolated nor normalized.
+    processed_transmission_table : np.ndarray
         A 2D array where the first col is wavelengths (Angstrom) and the second col is transmission values.
-    normalized_transmission_table : np.ndarray
-        A 2D array of wavelengths and normalized transmissions.
+        This table is both interpolated to the _wave_grid and normalized to calculate phi_b(λ).
     """
 
     def __init__(
         self,
         survey: str,
-        label: str,
+        filter_name: str,
+        delta_wave: Optional[float] = 5.0,
+        trim_quantile: Optional[float] = 1e-3,
         table_path: Optional[str] = None,
         table_url: Optional[str] = None,
         units: Optional[Literal["nm", "A"]] = "A",
```

Comment:
Should we just use astropy units here so we can use all their conversions?

Reply:
I think we've decided to not do conversions at this stage. The problem is the LSST filters online are in "nm", so we would have to do the conversion offline instead of downloading it directly

Reply:
Referenced in #113